src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Extensions/FileExtensionTests.cs

```diff
@@ -14,12 +14,16 @@ public class FileExtensionTests
         private const string PublicationSlug = "publication-slug";
         private const string ReleaseSlug = "release-slug";
 
-        private readonly File _file = new File
+        private readonly ReleaseFile _releaseFile = new ReleaseFile
```

Comment:
Everything for ReleaseFile to do with testing GetFileInfo can go in ReleaseFileExtensionTests